scripts/reportportal/rp_manual_reporter/collector.py

```
@@ -0,0 +1,476 @@
+    placeholder_files = scan_placeholder_tests(tests_dir=tests_dir)
+    details: list[PlaceholderTestDetail] = []
+
+    for placeholder_file in placeholder_files:
+        file_path = Path(placeholder_file.file_path)
+        full_path = tests_dir.parent / file_path
+
+        try:
+            source = full_path.read_text(encoding="utf-8")
+            tree = ast.parse(source=source)
+        except (OSError, SyntaxError) as exc:
+            LOGGER.warning(f"Failed to parse {full_path}: {exc}")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)